app/v2/models/response/retrieveForeignPropertyPeriodSummary/foreignNonFhlProperty/ForeignNonFhlPropertyRentIncome.scala

```diff
@@ -14,12 +14,14 @@
  * limitations under the License.
  */
 
-package v2.models.response.retrieveForeignPropertyPeriodSummary.foreignProperty
+package v2.models.response.retrieveForeignPropertyPeriodSummary.foreignNonFhlProperty
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.{JsPath, Json, Reads, Writes}
 
-case class ForeignPropertyRentIncome(rentAmount: BigDecimal)
+case class ForeignNonFhlPropertyRentIncome(rentAmount: Option[BigDecimal])
 
-object ForeignPropertyRentIncome {
-  implicit val format: OFormat[ForeignPropertyRentIncome] = Json.format[ForeignPropertyRentIncome]
+object ForeignNonFhlPropertyRentIncome {
+  implicit val writes: Writes[ForeignNonFhlPropertyRentIncome] = Json.writes[ForeignNonFhlPropertyRentIncome]
+  implicit val reads: Reads[ForeignNonFhlPropertyRentIncome] =
+    (JsPath \ "rentAmount").readNullable[BigDecimal].map(rentAmount => ForeignNonFhlPropertyRentIncome(rentAmount))
```

Comment:
same as earlier. if I'm wrong and it needs to be in there, these comments can be ignored.